samples/web/asp-mvc-injecting-bus/Version_5/WebApplication/Global.asax.cs

```diff
@@ -24,6 +24,8 @@ protected void Application_Start()
 
             BusConfiguration busConfiguration = new BusConfiguration();
             busConfiguration.EndpointName("Samples.MvcInjection.WebApplication");
+            // ExistingLifetimeScope() ensures that IBus is added to the container as well,
+            // allowing you to resolve IBus from your MVC controllers.
```

Comment:
Kill of the "from  your mvc controllers. That is a single usage. Not the only reason